config/app.yaml

```diff
@@ -5,7 +5,7 @@ pdc_root_dir: "data/{}/runfolders/"
 dsmc_log_dir: "logs/"
 job_timeout: "48h"      # maximum run-time for a job
 job_ttl: "72h"          # maximum time to keep a job in the queue
-job_result_ttl: "-1"    # maximum time to keep job result; -1 never expires
+job_result_ttl: "48h"   # maximum time to keep job result
```

Comment:
I had some issues with the test library not fetching the jobs properly and I experimented with changing this setting. Right now, I can't remember if this had something to do with the resolution though.. 🤔 Nevertheless, for maintainability, I would argue that it's probably wiser to have a maximum limit rather than keeping jobs indefinitely as, for various reasons, stuff will surely accumulate over time. Could perhaps be more than 48h but I think it should at least be set to _something_

Reply:
Yeah, I agree. I skimmed through some successful verify workflows and it looks like the longest take about 12 hours. So 48 h is probably a sound limit. I was mostly curious :)